<launch>
	<arg name="config_path" default = "$(find tfsensors)/config/config.yaml" />
	<node name="tf_publisher" pkg="tfsensors" type="tf_broadcaster" output="screen">

    	<param name="config_file" type="string" value="$(arg config_path)" />

	</node>
</launch>

